app/views/point_detail_comments/_show.html.erb

```diff
@@ -11,6 +11,6 @@
       <%= comment.created_at %>
     </div>
   </header>
-  <div class="uk-comment-body"><%= comment.comment %></div>
+  <div class="uk-comment-body" style="white-space: pre-wrap;"><%= comment.comment %></div>
```

Comment:
I know this right now is specific to this part of the app, but I'd still avoid using inline styles if possible.

Reply:
Good idea! 
I moved the CSS to a separate rule!